server/lib/tuist/qa/simulator_controller.ex

```diff
@@ -0,0 +1,121 @@
+defmodule Tuist.QA.SimulatorController do
+  @moduledoc """
+  Controller for interacting with iOS Simulator via xcrun simctl.
+  """
+
+  alias Tuist.QA.SimulatorDevice
```

Comment:
I think this can be 
```suggestion
  alias Tuist.QA.Simulators.Device
```

Or even better. We can have:

```
Tuist.Simulators
Tuist.Simulators.Device
```